Cargo.toml

```diff
@@ -19,9 +19,11 @@ edition = "2018"
 
 [dependencies]
 jobserver = { version = "0.1.16", optional = true }
+which = { version = "^4.0", optional = true }
```

Comment:
I'd ideally like to keep the deps of this crate pretty `slim`, and with `env::split_path` could  the `which` crate be replaced with an iteration over `PATH`?

Reply:
OK, I'll see what I can do... BTW, is it OK to keep it in cc-test/Cargo.toml?

Reply:
The trouble is that on Windows `which(x)` have to check for multiple file extensions (customarily listed in %PATHEXT% variable). So here is the question. Is it OK if the replacement subroutine won't be general-purpose `which`, but tailored for this specific use case? More specifically it would check for single `.exe` extension.

Reply:
oh yeah keeping it for `cc-test` is fine, but I think that this is only used for `nvcc`, right? If that's the case is `nvcc.exe` always the target? Or is it  frquently wrapped with other things like `.cmd`? (if it's always `.exe` we can just hardcode that)

Reply:
Checking for `.exe` alone is sufficient for the `cudart` purposes, totally. The concern was rather that if you add your own `which` subroutine, you run a risk that somebody else will want to use it, and then they are likely to have certain expectations. And the question was if it would be appropriate to curb these possible expectations. Answer appears to be "yes," and dependency on `which` crate is now removed.

Once again I want to point out that Rust is not my 1st-choice language, so that if you have any styling or any other nits, don't hold back:-)

Reply:
> Checking for `.exe` alone is sufficient for the `cudart` purposes, totally.

Just in case, why "totally." `.exe` takes precedence in PATH evaluation, so that you can't have a wrapper script called `nvcc.cmd` at the same location as `nvcc.exe` and keep using `nvcc` as command, you would have to call it explicitly as `nvcc.cmd`. In which case you would have to set NVCC environment variable if you want to use it with `cc-rs` anyway.